### ISSUE TITLE
Minor issue with git ID mismatch, rpm `Lmod-6.5.13-30_79b10ce_cm7.0_el7.noarch`

### ISSUE THREAD

**Issue by fgeorgatos** [plabrop/bic#56]
it's more of decorative, but I thought it was fixable, no?!

```
[fgeorgatos@node001 ~]$ ml --gitversion
git-8a2acb6
[fgeorgatos@node001 ~]$
[fgeorgatos@node001 ~]$ rpm -qa|grep -i lmod
Lmod-6.5.13-30_79b10ce_cm7.0_el7.noarch
```

**Comment by fgeorgatos** [comment]
for the record, this is just a filename issue, the reported `--gitversion` is the Lmod expected one

**Comment by fgeorgatos** [comment]
is this coming from some internal Bright process, which affects the filename on the basis of an internal commit tree? (I think we discussed it and it seems to be the case)

**Comment by fgeorgatos** [comment]
this is fixed in #63, no reason to keep this open
